lib/gateways/snapshot-gateway.js

```diff
@@ -80,20 +80,24 @@ class SnapshotGateway {
   async save({ snapshot }) {
     if (!snapshot) throw new ArgumentError('snapshot cannot be null.');
 
+    const updateExpression = ['set assets = :a', 'vulnerabilities = :v'];
+
+    const expressionAttributeValues = {
+      ':a': snapshot.assets,
+      ':v': snapshot.vulnerabilities
+    };
+
+    if (snapshot.notes) {
+      updateExpression.push('notes = :n');
+      expressionAttributeValues[':n'] = snapshot.notes;
+    }
```

Comment:
If the snapshot already has notes, and then they are deleted does this save the fact that they were deleted - or are they just ignored? (I'm also not sure if that scenario could ever happen - so maybe we don't need to worry about this?)

Reply:
We're not going to edit snapshots at this stage

Reply:
Ok, I'm cool with not overcomplicating it then at this stage - seems like something that is not going to happen! 👍